links/sm.json

```diff
@@ -0,0 +1,63 @@
+[
+{

```

Comment:
Either put these two on the same line or indent everything in the array one more step